arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java

```diff
@@ -202,8 +214,154 @@ public ComputationResponse compute(MixedRequest request, CollectionReference col
         return computationResponse;
     }
 
+    public Hits getSearchHits(MixedRequest request, CollectionReference collectionReference, Boolean flat, UriInfo uriInfo, String method) throws ArlasException {
```

Comment:
I suppose you only moved the code and didn't change anything?

Reply:
Indeed, you can compare from the original REST services.